test/rpc-test.js

```diff
@@ -80,13 +80,19 @@ describe('RPC', function() {
     assert(await nclient.execute('help', []));
     assert(await wclient.execute('help', []));
 
-    await assert.asyncThrows(async () => {
+    await assert.rejects(async () => {
       await nclient.execute('help', ['getinfo']);
-    }, 'getinfo');
+    }, {
+      name: 'Error',
+      message: /^getinfo/
```

Comment:
Also interesting, I don't think nodejs `assert.rejects()` accepts a regex for `message`:

https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message

another +1 bonus for b-suite dependencies!

Reply:
Documentation for `assert.rejects` it isn't exactly clear. Regexp are excepted for messages there too.

Reply:
Here are some examples of what I found that work and do not work.

```js
const assert = require('bsert');
//const assert = require('assert');

// Good (asserts)                                                                                                                                                                                                  
(async () => {
  assert.rejects(async () => {
    throw new Error('foo bar baz');
  }, {
    name: 'Error',
    message: /^foo baz/
  });

  assert.rejects(async () => {
    throw new Error('foo baz bar');
  }, {
    name: 'Error',
    message: 'foo bar baz'
  });

  assert.rejects(async () => {
    throw new Error('foo');
  }, /bar/);
})();


// Bad (does not assert)                                                                                                                                                                                           
(async () => {
  assert.rejects(async () => {
    throw new Error('foo');
  }, Error, 'bar');

  assert.rejects(async () => {
    throw new Error('foo');
  }, 'bar');

  assert.rejects(async () => {
    throw new Error('foo');
  }, Error, /bar/);
})();
```